build_runner/test/generate/build_test.dart

```diff
@@ -94,22 +96,44 @@ void main() {
       test('multiple mixed build actions', () async {
```

Comment:
nit: append " with custom build config" to the test name

Reply:
Done